packages/api-explorer/src/EndpointErrorBoundary.jsx

```diff
@@ -18,17 +18,14 @@ class EndpointErrorBoundary extends React.Component {
 
   render() {
     if (this.state.error) {
-      const mailTo = (
-        <a href="mailto:support@readme.io?subject=API Explorer Error">support@readme.io</a>
-      );
-
       return (
         <div className="hub-reference-section">
           <div className="hub-reference-left" style={{ paddingLeft: '2%' }}>
             <h3>
               There was an error rendering this endpoint. If you are the owner of this project
-              please contact
-              {` ${mailTo} with the following error:`}
+              please contact&nbsp;
```

Comment:
These `mailTo` changes are because when errors were present, the copy for this header was:

> There was an error rendering this endpoint. If you are the owner of this project please contact [Object object] with the following error: